sim/src/test/scala/midasexamples/TutorialSuite.scala

```diff
@@ -259,7 +259,7 @@ class PrintfCycleBoundsTestBase(startCycle: Int, endCycle: Int) extends Tutorial
       s"+print-start=${startCycle}",
       s"+print-end=${endCycle}"
     )) {
-  checkPrintCycles("synthprinttest.out0", startCycle, endCycle, linesPerCycle = 5)
+  checkPrintCycles("synthprinttest.out0", startCycle, endCycle, linesPerCycle = 4)
```

Comment:
Is there a change in the output of printfs?

Reply:
I removed one print in RTL for the tests. No change for any `printf` instance.